themes/reactos/layouts/partials/recent_posts.html

```diff
@@ -38,6 +38,11 @@ <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
 			</section>
 
 			{{ end }}
+
+		<ul class="pager">
+			<li class="next"><a href="https://pr71.web-content.reactos.org/news">Older posts →</a></li>
```

Comment:
```suggestion
			<li class="next"><a href="/news">Older posts →</a></li>
```

Reply:
Fixed. Thanks @ColinFinck.
Now it looks even better and still works for me.